packages/frontend/core/src/components/blocksuite/block-suite-header/menu/index.tsx

```diff
@@ -79,6 +83,17 @@ export const PageHeaderMenuButton = ({
   const { importFile } = usePageHelper(docCollection);
   const { setTrashModal } = useTrashModalHelper(docCollection);
 
+  const workbench = useService(WorkbenchService).workbench;
+  const view = useService(ViewService).view;
+
+  const openSidePanel = useCallback(
```

Comment:
This useCallback has no effect because it always returns a new function after calling

Reply:
Good catch! I have updated code that use another `useCallback` to create click handle functions